packages/browser-integration-tests/suites/public-api/captureException/errorEvent/test.ts

```diff
@@ -0,0 +1,25 @@
+import { expect } from '@playwright/test';
+import type { Event } from '@sentry/types';
+
+import { sentryTest } from '../../../../utils/fixtures';
+import { getFirstSentryEnvelopeRequest } from '../../../../utils/helpers';
+
+sentryTest('should capture an ErrorEvent', async ({ getLocalTestPath, page, browserName }) => {
+  // On Firefox, the ErrorEvent has the `error` property and thus is handled separately
+  if (browserName === 'firefox') {
+    sentryTest.skip();
+  }
```

Comment:
Do we just want to add the FF behaviour to this test?

Reply:
I leave it up to you. If you think it's valuable we can add a test - otherwise it's fine too.